### PR TITLE
[Ssd13xx] Add support for double-byte character 16 size font display

### PR DESCRIPTION
**Pull request by yureng**
## Motivation and Context

Need to support Asian language text display on Ssd13xx.

## How Has This Been Tested?

ESP32-S  and Ssd1306，CLR Version: 1.8.0.291

## Screenshots
Screenshots

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [x] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.IoT.Device?pullRequest=497) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.IoT.Device?pullRequest=497) before we can accept your contribution.<br/><br/>:x: yureng [sign now](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.IoT.Device?pullRequest=497)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nanoframework/nanoFramework.IoT.Device?pullRequest=497) it.</sub>

File: devices/Ssd13xx/Ssd13xx.cs [hunk 1]
```diff
@@ -394,6 +394,46 @@ byte[] GetTextBytes(string text)
...
+            if (Font.Width == 16)
+            {
+                var charCount = (text.Length * 2);
+                bitMap = new byte[charCount * Font.Height * Font.Width / 16];
+
+                byte[] characterMap = null;
+                for (int i = 0; i < charCount; i++)
+                {
+                    var even = (i == 0 || i % 2 == 0);
+                    if (even)
```
Comment (josesimoes) [inline devices/Ssd13xx/Ssd13xx.cs:416]: Please add open and close brackets.

File: devices/Ssd13xx/Ssd13xx.cs [hunk 2]
```diff
@@ -394,6 +394,46 @@ byte[] GetTextBytes(string text)
...
+                        characterMap = Font[text[i / 2]];
+
+                    var list = new System.Collections.ArrayList();
+                    for (int idx = (even ? 0 : 1); idx < characterMap.Length; idx += 2)
```
Comment (josesimoes) [inline devices/Ssd13xx/Ssd13xx.cs:420]: Same here

**Review by josesimoes — CHANGES_REQUESTED** [review]

**Comment by josesimoes** [comment]
Nice improvement! 👍🏻

**Review by Ellerbach — COMMENTED** [review]
Nice addition! Can you please as well adjust the README and add your nice picture in there?
For the code, Jose remarks has to be addressed.
Thanks!

**Comment by yureng** [comment]
> Nice addition! Can you please as well adjust the README and add your nice picture in there? For the code, Jose remarks has to be addressed. Thanks!



> 

Jose's remarks have been addressed.

I'm sorry! I can't seem to change the picture that the README has shown correctly. 
Also, my English is poor and I need to rely on Google Translate and can't work properly

**Review by josesimoes — APPROVED** [review]
You're doing great! So far nothing has prevented a good communication flow. 😃 
And I've just updated the readme with this usage sample and your image of the display.
Thanks for this nice improvement! 👏🏻

**Pull request merged**